build/cocos2d-win32.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
```

Comment:
Thanks. `MinimumVisualStudioVersion` is still `12.0.21005.1` but I'll change it anyway